tox.ini

```diff
@@ -0,0 +1,88 @@
+[tox]
+skipsdist=True
+skip_missing_interpreters = True
+envlist = fmt, lint, unit
+
+[vars]
+src_path = {toxinidir}/src/
+tst_path = {toxinidir}/tests/
+lib_path = {toxinidir}/lib/charms/nrpe/
+all_path = {[vars]src_path} {[vars]tst_path}
+
+[testenv]
+setenv =
+  PYTHONPATH = {toxinidir}:{toxinidir}/lib:{[vars]src_path}
+  PYTHONBREAKPOINT=pdb.set_trace
+  PY_COLORS=1
+  juju2: LIBJUJU="2.9.44"  # libjuju2
+  juju3: LIBJUJU="3.2.2"  # libjuju3
+passenv =
+  PYTHONPATH
+  CHARM_BUILD_DIR
+  MODEL_SETTINGS
+
+[testenv:fmt]
+description = Apply coding style standards to code
+deps =
+    # renovate: datasource=pypj
+    black==23.7.0
+    # renovate: datasource=pypi
+    ruff==0.0.287
+commands =
+    ruff --fix {[vars]all_path}
+    black {[vars]all_path}
+
+[testenv:lint]
+description = Check code against coding style standards
+deps =
+    # renovate: datasource=pypi
+    black==23.7.0
+    # renovate: datasource=pypi
+    ruff==0.0.287
+    # renovate: datasource=pypi
+    codespell==2.2.5
+commands =
+    codespell {toxinidir}
+    ruff {[vars]all_path}
+    black --check --diff {[vars]all_path}
+
+[testenv:unit]
+description = Run unit tests
+deps =
+    -r{toxinidir}/requirements.txt
+    # renovate: datasource=pypi
+    cosl==0.0.6
```

Comment:
What is this used for?

Reply:
right now it is not used so I should remove it. But this is to integrate cos lite stack into the charm so will be added in the future anyway. However, I'll remove it for now.